index.html

```diff
@@ -0,0 +1,60 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="mobile">
+        <div class="navigation">
+            <div class="section-1 section-time">
+                <a class="newton" href="">Newton Alumasa</a>
+                <input type="checkbox" class="navigation__checkbox" id="navi-toggle">
+                <label for="navi-toggle" class="navigation__button" id="navi-button">
+                    <span class="navigation__icon"></span>
+                </label>
+                <div class="navigation__background" id="navi-background"></div>
+                <nav class="bar" id="bar">
+                    <div class="bar1"><a href="#portfolio">Portfolio</a></div>
+                    <div class="bar2"><a href="#about">About</a></div>
+                    <div class="bar3"><a href="#contact">Contact</a></div>
+                    <div class="bar3"><a href="" aria-label="union"><img src="img/Union.png" alt="union"></a></div>
+                </nav>
+            </div>
+        </div>
+        <section class="content">
+            <aside class="sidebar">
+                <ul>
+                    <li><a href="" aria-label="github"><img src="img/Icon-GitHub.png" alt="github"/></a></li>
+                    <li><a href="" aria-label="linkedin"><img src="img/Icon-Linkedin.png" alt="linkedin"/></a></li>
+                    <li><a href="" aria-label="medium"><img src="img/Icon-Medium.png" alt="medium"/></a></li>
+                    <li><a href="" aria-label="angelist"><img src="img/Icon-Angelist.png" alt="angelist"/></a></li>
+                    <li><a href="" aria-label="twitter"><img src="img/Frame.png" alt="twitter"/></a></li>
+                </ul>
+            </aside>
+            <section class="content-1">
+                <div class="headline">
+                    <p>Hey There.<br>Am Newton</p>
+                </div>
+                <div class="headline-1">
+                    <p>I'm a Software Developer</p>
+                </div>
+                <div class="headline-2">
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, dont hestiate to contact me.</p>
```

Comment:
Great job @altontonn.🤝 

- [ ] Please pay attention to the font size and font weight of your two classes. headline-1 and headline-2. They don't follow the actual Figma design.
- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/176536502-40a6ca67-bd3e-4707-902c-d4d5a332893d.PNG)

- Your design
![Your design](https://user-images.githubusercontent.com/63917725/176536562-90bec0d1-c51a-44fc-8e29-03e54109df16.PNG)